students/WesleyM/Lesson4/mailroom2.py

```diff
@@ -0,0 +1,55 @@
+donors = [
+  {'name': 'Alice Adams Ron', 'donation': [20]} ,
+  {'name': 'Bob Be-Lake', 'donation': [100]},
+  {'name': 'Charles Cruz', 'donation': [30, 50, 10]},
+  {'name': 'Denise Dnice', 'donation': [10, 5]},
+  {'name': 'Edward Eduardo', 'donation': [25, 20, 20, 20, 10]}
+  ]
+ 
```

Comment:
I would suggest using a dict as your main data structure here rather than a list of dicts